client/js/nonprofits/donate/wizard/utils/parseDonateParams.ts

```diff
@@ -0,0 +1,11 @@
+// License: LGPL-3.0-or-later
+
+const url = require('url')
+
+export default function parseDonateParams(location:Location, app:Record<string, string>) {
+  const params = url.parse(location.href, true).query
+  params.hide_cover_fees_option = params.hide_cover_fees_option || app.hide_cover_fees_option
+  params.manual_cover_fees = params.manual_cover_fees || app.manual_cover_fees
+
+  return params;
+}
```

Comment:
I moved the common parts of the donate params parsing into one place. It's not the ultimate place this should go but at least it's not duplicated in multiple places.